core/roboconf-agent-monitoring/src/main/java/net/roboconf/agent/monitoring/internal/MonitoringTask.java

```diff
@@ -138,14 +155,14 @@ public void run() {
 				continue;
 
 			if( s.toLowerCase().startsWith( RULE_BEGINNING )) {
-				sections.add( sb.toString());
+				if(sb.length() > 0) sections.add( sb.toString());
```

Comment:
Is this really usefl to check if sb.length() is positive?
You append "\n" below, so, even if expandString return "", you will add a new section anyway.